src/_data/LiveData.js

```diff
@@ -56,15 +56,14 @@ export const changeLanguage = langCode => {
 };
 
 const initUnauthorized = async () => {
-
-};
-
-const initAuthorized = async (authData, store) => {
     api.getActiveSymbolsFull();
     api.getTradingTimes(new Date());
     api.getAssetIndex();
     api.getServerTime();
     api.getCandlesForLastNDays('R_100', 30);
```

Comment:
In fact, next thing will be to load TradingTimes on first TradingTimesContainer render.
Same for everything that we do not depend on. Though I think we depend on trading times :( and asset index and may be few more calls.